Les5-backend-java-pokemon-interface.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Je .iml bestand en je .idea map mag je in je .gitignore bestand zetten. Deze hoef je niet te uploaden naar github namelijk.

Reply:
Je out-map mag ook in de .gitignore